src/Sulu/Component/Content/Metadata/Parser/SchemaXmlParser.php

```diff
@@ -24,13 +24,13 @@ public function load(\DOMXPath $xpath, \DOMNode $contextNode)
         $allOfNode = $xpath->query('x:allOf', $contextNode)->item(0);
         $allOfs = [];
         if ($allOfNode) {
-            $allOfs = $this->loadAllOfs($xpath, $allOfNode, $contextNode);
+            $allOfs = $this->loadAllOfs($xpath, $allOfNode);
```

Comment:
not sure here as the params are: `(\DOMXPath $xpath, \DOMNode $contextNode)`

Reply:
I think this change is correct